README.md

```diff
@@ -1,4 +1,4 @@
-# Module Source Imports
+# Import Source Reflection
```

Comment:
Let's rename to `Source Phase Imports`?

Reply:
I've come around on the `Reflection` term finally - in fact the `ModuleSource` option is defined in the `Reflection` section now. The `ModuleSource` once it has bindings will truly be a reflective data type. `Import Source Reflection` allows us to somewhat salvage the original name.

Reply:
I think it's confusing still. I'd prefer "phase".

Reply:
Sure, `Source Phase Imports` makes sense, and but strictly speaking, we are treating `ModuleSource` as a reflection object now.

Reply:
I prefer to reserve `Reflection` for the proposal that makes module sources reflect the parsed bindings. I propose that `Source Phase Imports` is clearer for this layer.